interface/patient_file/summary/demographics.php

```diff
@@ -749,78 +751,110 @@ function setMyPatient() {
 if ($GLOBALS['patient_id_category_name']) {
     $idcard_doc_id = get_document_by_catg($pid, $GLOBALS['patient_id_category_name']);
 }
+$res=sqlStatement("SELECT option_id,activity FROM list_options WHERE list_id='patient_file_menu' ");
+
+while ($row = sqlFetchArray($res)) {
+    $patient_file_menu[$row['option_id']]=$row['activity'];
+}
+
+// Use a json file to build menu
+// Collect the selected menu of user
+$patientMenuRole = PatientMenuRole::getPatientMenuRole();
+// Load the selected menu
+if (preg_match("/.json$/", $patientMenuRole)) {
+    // load custom menu (includes .json in id)
+    $menu_parsed = json_decode(file_get_contents($GLOBALS['OE_SITE_DIR'] . "/documents/custom_menus/patient/menus/" . $patientMenuRole));
```

Comment:
should be patient_menus and not patient/menus in above path

Reply:
ok